### PR TITLE
Feature/Question section

### PR DESCRIPTION
**Pull request by smallpaes**
# Related Issue / Function
- 使用者可以在 FAQ 頁面查看所有常見問題與平台回覆
- 更新 README
- 更新 User Profile Form

# Proposed Changes
- 新增 Faq view
- 更新 Footer 連結
- 新增 Tooltip component
- 在 UserProfileForm 在新增地址欄位 Tooltips
- 更新根路徑 README 內容、刪除 client 資料夾中的 README 檔案

# Additional Info
- Any additional information

# Checklist
- [x] Function testing
- [ ] Run testing / Go through checklist
- [x] Update Trello / Checklist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,206 @@
...
+
+
+## How to use
+
+### Visitor
+
+As a visitor, you may: 
+1. Learn more about the platform and how it works
+2. Check out all the restaurants currently serving on the platform
+3. Check out the detail of all the meal plans available
```
Comment (windate3411) [inline README.md]: 感覺改為 subscription plans會再更恰當些

File: README.md [hunk 2]
```diff
@@ -1,17 +1,206 @@
...
+4. Visit our F.A.Q page for all the questions and response
```
Comment (windate3411) [inline README.md]: L91 Visit our F.A.Q page for all the questions and response 看需不需要改為
Visit our F.A.Q page for all the frequently asked questions and answers

File: README.md [hunk 3]
```diff
@@ -1,17 +1,206 @@
...
+
+As a Nextmeal member, you may:
+- Log in to your Nextmeal user account
+  - email: user1@example.com
+  - password: Nextmeal!
+- Subscribe a meal plan--10 meals or 20 meals per month--and pay it online with credit card
+  - Credit card number(testing): 4000-2211-1111-1111
+  - Credit card expiration date: Any random date
+  - Credit card security code: Any random three numbers set
+- Choose between two restaurants we provide every day and order the meal in advance
```
Comment (windate3411) [inline README.md]: L105 看需不需要加入距離相關的描述 (e.g Choose between two restaurants we provide within 500 meters every day...

File: README.md [hunk 4]
```diff
@@ -1,17 +1,206 @@
...
+### Restaurant Owner
+
+As a restaurant owner, you may:
+- Log in to your Nextmeal owner account
+  - email: user2@example.com
+  - password: Nextmeal!
+- Edit your restaurant info
+- Create and edit your restaurant menu
+- See the meal currently serving
+- Update the meal and amount you would like to serve next week
```
Comment (windate3411) [inline README.md]: L120 可以加入關於時間的限制說明 (週六後便無法修改)

File: README.md (hunk 1 above)
Comment (smallpaes) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (smallpaes) [reply README.md]: Added

File: README.md (hunk 4 above)
Comment (smallpaes) [reply README.md]: Added

**Pull request merged**